docs/getting_started.md

```diff
@@ -12,7 +12,7 @@ The required versions of MMCV and MMDetection for different versions of MMDetect
 
 | MMDetection3D version | MMDetection version |    MMCV version     |
 |:-------------------:|:-------------------:|:-------------------:|
-| master              | mmdet>=2.10.0       | mmcv-full>=1.2.4, <=1.4|
+| master              | mmdet>=2.10.0       | mmcv-full>=1.3.1, <=1.4|
```

Comment:
Please also modify this [file](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/__init__.py) correspondingly.

Reply:
Done.

Reply:
1.3.2